ThroughputTest/Program.cs

```diff
@@ -0,0 +1,65 @@
+﻿//---------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.  
+//
+// THIS CODE AND INFORMATION ARE PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND, 
+// EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED WARRANTIES 
+// OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE. 
+//---------------------------------------------------------------------------------
+
+namespace ThroughputTest
```

Comment:
>ThroughputTest [](start = 10, length = 14)

This is a generic comment on comments in the code or the lack there off. I think the code is well done and read able but for people who want to modify it a few key comments maybe in settings, program, performance app and Receive and Send experiment may help?